### PR TITLE
Cleanup core with baseline / information model

### PR DESCRIPTION
**Pull request by mlagally**
This PR aligns the language in the first half of the document with the new name "baseline profile".

It also carries over improvements from https://github.com/w3c/wot-profile/pull/93, 
which have been adapted to the new document structure.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-profile/pull/203.html" title="Last updated on May 25, 2022, 2:59 PM UTC (8a61bfe)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-profile/203/ec871ab...8a61bfe.html" title="Last updated on May 25, 2022, 2:59 PM UTC (8a61bfe)">Diff</a>

### CODE REVIEW

**Review by benfrancis — CHANGES_REQUESTED** [review]
Whilst I welcome the slight reduction in constraints, in my view these changes do not go far enough to address concerns with this section of the specification.

Please see my comments in https://github.com/w3c/wot-profile/issues/10#issuecomment-1129296959 where I suggest a more significant re-working of this section to remove unnecessary constraints and focus on the agreed requirements of the specification.

Examples:
- There is no need for title, description, base, id or version to be mandatory anywhere
- There is no need to constrain id to be a URN rather than anyURI
- There is no need to arbitrarily constrain the length of the title (or any) member
- There is no need to require arrays where a simple string would suffice
- There is no need to constrain the types of enums
- The recommendation to use an empty string for undefined values is undesirable
- Requirements about email addresses for developers and support are arbitrary and unnecessary
- There are valid use cases for properties without a type member (e.g. endpoints which resolve to a media file such as an image, audio or video file)
- There is no need to arbitrarily limit the depth of nested data structures
- There is no need to disallow the const keyword
- The constraint on the number of forms per interaction affordance and constraints on the values of op members are actively harmful to interoperability
- There's no need to disallow oneOf for properties
- Disallowing the use of uriVariables is actively harmful for interoperability
- There is no need to constrain the use of security fields and scopes

**Comment by mlagally** [comment]
@benfrancis 
Thanks for your review and the list of your concerns.

This MR does not try to solve these problems, it's scope is just an alignment of language across the entire document.
I'm aware that we need to talk about these constraints and will address your concerns in one of the next profile calls, where I hope you can attend for a live discussion.

Following this clarification and alignment there will be other MRs.

**Review by benfrancis — COMMENTED** [review]
I'm sorry I'm unable to attend the WoT Profile meeting today.

I don't agree with the content of the content of the Baseline Information Model section, but given this PR is an improvement over what's currently there and will help to make this section less confusing as we continue the discussion, I would be OK with merging this PR if an Editor's Note is added to say the whole of section 6.2 is "at risk", perhaps linking to #10 for further discussion.

To be clear, we don't intend to implement the assertions in section 6.2 as currently written in WebThings.

File: index.html [hunk 1]
```diff
@@ -925,90 +929,126 @@ <h3>Mandatory fields</h3>
...
-                <td>human readable documentation</td>
-              </tr>
-              <tr class="rfc2119-table-assertion" id="profile-5.1.2.1-thing-mandatory-fields-1--security">
-                <td>security</td>
-                <td>array of string</td>
-                <td>simplified handling</td>
+              <tr>
+                <td>profile</td>
+                <td><code>string</code></td>
+                <td>value: "https://www.w3.org/2021/wot/profile/baseline-json-http11"</td>
```
[the remark this answers is not in the document]
Comment (mlagally) [reply index.html]: done

**Pull request merged**